packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -315,13 +317,26 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					"Unexpected error: No Vite HTTP server"
 				);
 
+				const inputInspectorPort = await getInputInspectorPortOption(
+					pluginConfig,
+					viteDevServer
+				);
+
```

Comment:
Not sure it's right to run this on restarts i.e. when the Worker config changes etc.. Isn't there a risk that it assigns a new port each time because it's already in use? Could you use `getInputInspectorPortOption` in the `if` block and reuse the returned value in the `else` block?

Reply:
I don't think that that's the right call 🤔 

Note that this runs also when you make changes in the `vite.config.ts`, if I were to make a change there during development to the `inspectorPort` itself (say from it being undifined to a specific port for example) I would definitely want this change to picked up and reflected in my dev server, wouldn't I?
(PS: the above doesn't even work nicely right now 😢, I'm looking into why that is)

Reply:
oof.... the above does not work properly because miniflare does not update the inspector port on `.setOptions` 😓  I will look into that, but if that's ok by you I would do it in a followup PR? 🙏

Reply:
Good thinking about needing to support changing the `inspectorPort` in the Vite config. Yes, happy for this to be merged as is and for that to be investigated in a follow-up PR.